### DISCUSSION TITLE
Open scenes are displayed only when the corresponding scene editor is opened

### DISCUSSION THREAD

**Discussion by L2750558108** [godotengine/godot-proposals#6009] [Editor]
### Describe the project you are working on

no

### Describe the problem or limitation you are having in your project

no

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

The operating logic of the editor will be better

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

For example, when "2D" or "3D" is selected, the red circle position shows the currently open scene, and when "Script" or "AssestLib" is selected, the tab of the red circle position is hidden

![image](https://user-images.githubusercontent.com/119792950/209661265-d1b671c2-6536-48a4-8e19-8d5c63e74ec1.png)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

no

### Is there a reason why this should be core and not an add-on in the asset library?

no

**Comment by Calinou** [comment]
I think this is done by design, at least in the Script tab. The currently selected scene tab affects autocompletion results for node paths.

**Reply by L2750558108** [reply]
That can hide the scene tab in AssestLib

**Reply by KoBeWi** [reply]
This could be an option for main screen plugins. I have a few plugins that don't need scene tabs.
